packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -664,54 +751,58 @@ Slider.propTypes = {
    * Should not be equal to max.
    */
   min: PropTypes.number,
+  /**
+   * Name attribute of the hidden `input` element.
+   */
+  name: PropTypes.string,
   /**
    * Callback function that is fired when the slider's value changed.
+   *
+   * @param {object} event The event source of the callback
+   * @param {any} value The new value
    */
   onChange: PropTypes.func,
   /**
-   * Callback function that is fired when the slide has stopped moving.
+   * Callback function that is fired when the mouseup is triggerd.
+   *
+   * @param {object} event The event source of the callback
+   * @param {any} value The new value
    */
-  onDragEnd: PropTypes.func,
+  onChangeCommitted: PropTypes.func,
   /**
-   * Callback function that is fired when the slider has begun to move.
+   * @ignore
    */
-  onDragStart: PropTypes.func,
+  onMouseDown: PropTypes.func,
   /**
-   * The granularity the slider can step through values.
+   * The stepper orientation (layout flow direction).
    */
-  step: PropTypes.number,
+  orientation: PropTypes.oneOf(['horizontal', 'vertical']),
   /**
-   * @ignore
+   * The granularity the slider can step through values.
+   * When step is `null`, users can only slide the thumbs onto marks.
```

Comment:
```suggestion
   * When step is `null`, the thumb can only be slid onto marks provided with the `marks` prop.
```

Reply:
However when not null, it can only be slid onto values according to `step`. I would argue it should be the union of these, so that `37` can always be selected in the demo. Providing a marked value, but not being able to select it doesn't seem right (in that example at least).

Reply:
cf https://github.com/mui-org/material-ui/pull/15703#discussion_r289658384